frontend/src/index-breakout.tsx

```diff
@@ -4,38 +4,35 @@ import { App } from "./components";
 import "./styles/index.css";
 
 declare const listen: Function;
+declare const subscribe: Function;
+declare const emit: Function;
 
 var rootEditorElement;
 
-/**
- * Initializes the base markup before page is ready. This is not part of the API, and called explicitly at the end of this module.
- */
-function init() {
-  rootEditorElement = document.createElement("div");
-  rootEditorElement.innerHTML = `<h3>Breakout: should be replaced by React</h3>`;
-  document.body.appendChild(rootEditorElement);
-  ReactDOM.render(
-    <React.StrictMode>
-      <App />
-    </React.StrictMode>,
-    rootEditorElement
-  );
-}
-
 export const createBreakoutApp = () => {
-  listen("sfcc:ready", () => {
-    init();
+  subscribe("sfcc:ready", () => {
+    rootEditorElement = document.createElement("div");
+    rootEditorElement.innerHTML = `<h3>Breakout: should be replaced by React</h3>`;
+    document.body.appendChild(rootEditorElement);
+    ReactDOM.render(
+      <React.StrictMode>
+        <App />
+      </React.StrictMode>,
+      rootEditorElement
+    );
+    const openButtonEl = document.querySelector(".ix-selection");
+    if (openButtonEl) {
+      openButtonEl.addEventListener("click", handleSelect);
+    }
```

Comment:
I know this is an example so maybe you were going to do this anyway, but I think we should use props for this event handler. i.e: 

```suggestion
    ReactDOM.render(
      <React.StrictMode>
        <App onValueSet={handleSelect} />
      </React.StrictMode>,
      rootEditorElement
    );
```